receiver/purefareceiver/README.md

```diff
@@ -0,0 +1,43 @@
+# Pure Storage Flash Array Receiver
+
+| Status                   |                     |
+| ------------------------ |---------------------|
+| Stability                | [in-development]    |
+| Supported pipeline types | metrics             |
+| Distributions            | [contrib]           |
+
+The Pure Storage Flash Array receiver, receives metrics from Pure Storage internal services hosts.
+
+Supported pipeline types: metrics
+
+## Configuration
+
+The following settings are required:
+
+ -  
+
+The following setting are optional:
+
+ - 
+ 
+Example:
+
+```yaml
+receivers:
+  purefa:
+    endpoint:
+    arrays:
+      address:
+      token:
+    settings:
+      metrics_array_collection_interval:
+      metrics_host_collection_interval:
+      metrics_volume_collection_interval:
+      metrics_pod_collection_interval:
+```
```

Comment:
there are 5 metric types per [the OpenMetrics exporter](https://github.com/PureStorage-OpenConnect/pure-fa-openmetrics-exporter/blob/master/README.md) `array`, `host`, `volume`, `pods`, `directories`